.github/workflows/build-wasm.yml

```diff
@@ -46,4 +46,4 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: build
-          args: -p prisma-fmt --release --target=wasm32-unknown-unknown
+          args: -p prisma-fmt --lib --release --target=wasm32-unknown-unknown
```

Comment:
TODO: use the script there, or reflect it

Reply:
After some more reflection: let's leave that to the next PR, NPM automation is not trivial.